spidermon/contrib/scrapy/monitors.py

```diff
@@ -0,0 +1,146 @@
+from spidermon import monitors, MonitorSuite, Monitor
+from spidermon.exceptions import NotConfigured
+from ..monitors.mixins.spider import SpiderMonitorMixin
+
+SPIDERMON_MIN_ITEMS = 'SPIDERMON_MIN_ITEMS'
+SPIDERMON_MAX_ERRORS = 'SPIDERMON_MAX_ERRORS'
+SPIDERMON_EXPECTED_FINISH_REASONS = 'SPIDERMON_EXPECTED_FINISH_REASONS'
+SPIDERMON_UNWANTED_HTTP_CODES = 'SPIDERMON_UNWANTED_HTTP_CODES'
+
+
+class BaseScrapyMonitor(Monitor, SpiderMonitorMixin):
+    longMessage = False
+
+    def get_settings(self, name, default=None):
```

Comment:
I still think that using `custom_settings` to define particular values for each spider is more consistent with the way Scrapy uses for defining settings for each spider, but it is fine as it is.

Reply:
There is no tests for this method.

Reply:
I don't know if they are required either.
This method is not part of the public API, so I don't think we need to test it directly.

Reply:
It's tested because the other monitors relies on it, so it's called and tested. Please refer to the coverage.